project-reports/2023/2023-Q1-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,271 @@
+
+# Project Health
+
+-   Cactus and Weaver were merged to one project "Cacti."  The
+maintainers almost completed the first code merge and will release
+the first Cacti v2-alpha (tentative name) in Q1 2023.
+-   The maintainers are also discussing the new architecture of Cacti
+toward Cacti-V2 in order to combine the strengths of the original
+Cactus and Weaver to achieve a more flexible interoperability tool.
+-   Cacti has newly supported the interoperability with Iroha V2 and
+Ubiquity SDK.
+-   Weaver now supports organizational identity syncing and event
+pub/sub across Fabric networks, and atomic swaps involving
+ERC-standard tokens in Besu networks with Fabric and Corda
+networks.  These will be merged in Cacti V2-alpha release on Q1
+2023.
+
+# Required Information
+
+1.  <span style="color: rgb(68,68,68);"> <a href="https://wiki.hyperledger.org/display/TSC/Projects+have+two+quarters+to+comply+with+common+repo+structure?focusedCommentId=41591637#comment-41591637" rel="nofollow">Have you switched from master to main in all your
+repos</a> </span> ? **Yes**
+2.  <span class="placeholder-inline-tasks" style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(68,68,68);">
+<a href="https://tsc.hyperledger.org/repository-structure.html" class="external-link" rel="nofollow">Have you implemented the Common
+Repository Structure in all your repos</a> </span> </span> <span style="color: rgb(23,43,77);text-decoration: none;">? **Yes**</span>
+3.  <span style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(23,43,77);">Has your project implemented these
+inclusive language changes listed below to your repo? You can
+optionally
+<a href="https://github.com/petermetz/gh-action-dci-lint#usage" class="external-link" rel="nofollow">use the DCI Lint tool</a> to
+make this a recurring action on your repo. **Yes, we use DCI-Lint as
+part of our CI**</span>
+1.  master → main
+2.  slave → replicas
+3.  blacklist → denylist
+4.  whitelist → allowlist
+4.  <span style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(23,43,77);">Have you added an <a href="https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example" rel="nofollow">Inclusive Language Statement</a> to your project's
+documentation and/or Wiki pages?  **Yes**</span>
+
+# Questions/Issues for the TSC
+
+-   None at the moment.
+
+# Releases
+
+-   v1.1.x: to fix the GitHub workflow errors
+-   v1.1.3 =&gt;
+<a href="https://github.com/hyperledger/cactus/tree/v1.1.3" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.3</a>
+-   v1.1.2 =&gt;
+<a href="https://github.com/hyperledger/cactus/tree/v1.1.2" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.2</a>
+-   v1.1.1 =&gt;
+<a href="https://github.com/hyperledger/cactus/tree/v1.1.1" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.1</a>
```

Comment:
```suggestion
    -   v1.1.3 =&gt;
<a href="https://github.com/hyperledger/cactus/tree/v1.1.3" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.3</a>
    -   v1.1.2 =&gt;
<a href="https://github.com/hyperledger/cactus/tree/v1.1.2" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.2</a>
    -   v1.1.1 =&gt;
<a href="https://github.com/hyperledger/cactus/tree/v1.1.1" class="external-link" rel="nofollow">https://github.com/hyperledger/cactus/tree/v1.1.1</a>
```

Reply:
I think I addressed both of these in the latest push.